Project/webpack.config.js

```diff
@@ -12,6 +14,16 @@ const communicationIdentityClient = new  CommunicationIdentityClient(config.conn
 
 const PORT = process.env.port || 8080;
 
+
+const onSignalRegistrationTokenToAcsUserAccesTokenMap = new Map();
+
+const generateGuid = function () {
```

Comment:
I believe we already have a built in function for this

Reply:
yeah i just couldnt import it from utils cus that files already using import/export schema